src/types.rs

```diff
@@ -0,0 +1,16 @@
+use rust_decimal::Decimal;
```

Comment:
Having a `types.rs` file is a code smell that indicates a lack of domain oriented design-thinking.